cves/CVE-2015-6826.yml

```diff
@@ -43,11 +43,11 @@ announced_instructions: |
 
 
   Please enter your date in YYYY-MM-DD format.
-announced_date:
```

Comment:
no reported date?

Reply:
Not that I could find. Other than the fix commit itself, I couldn't really find anything anywhere that referenced the vulnerability. This `announced_date` is just when the new version with the fix was released.